XRTK-Core/Packages/com.xrtk.core/Tests/Services/ITestExtensionDataProvider1.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace XRTK.Tests.Services
 {
-    internal interface ITestExtensionDataProvider1 : IMixedRealityExtensionDataProvider, ITestService { }
+    internal interface ITestExtensionDataProvider1 : IMixedRealityExtensionDataProvider { }
```

Comment:
Why did we remove the ITestService interface from all the classes?

Reply:
Because ITestService directly inherits from IMixedRealityService
```
ITestService : IMixedRealityService
```

This is an extension service, so should inherit from 'IMixedRealityExtensionService', else it is not a valid test of an ExtensionService.

Reply:
Even crazier, was the fact the data providers were configured to inherit directly from IMixedRealityService, which is not a real-world test case.

Reply:
Ah I see, it should probably be inheriting from a `ITestDataProvider` instead of a service. great catch!

Reply:
Except it shouldn't be inheriting from something that isn't for "Extensions", hence 

ExtensionService -> ExtensionDataProvider.

Shouldn't have anything to do with the base versions of Data Provider, unless we are dropping the "Extension" segment for Data Providers, which wouldn't be the worst thing.  A data provider is just a data provider, it's the Service/System that is the extension.

Reply:
I mean there's really no difference between the extensions vs non extensions at the moment (Maybe in the future?), Which is why I'm confused about how and why we might be seeing different results.

Reply:
Extension Services to be removed in a separate PR